Sources/ArgumentsUtils.swift

```diff
@@ -88,6 +90,11 @@ enum TemplateError: Error, CustomStringConvertible {
       "or use --templatePath to specify a template by its full path."
     case .templatePathNotFound(let path):
       return "Template not found at path \(path.description)."
+    case .noTemplateFound:
+      return "A template must be chosen either using the named option --template or the path option --templatePath."
+    case let .multipleTemplateOptions(path, name):
+      return "You need to choose EITHER a named template (option --template) OR a template path (option --templatePath). " +
```

Comment:
Dammit I missed that on first review, but also: 
"--template option" not "option --template"